torch/__init__.py

```diff
@@ -751,7 +751,8 @@ def manager_path():
     obj = getattr(_C._VariableFunctions, name)
     obj.__module__ = 'torch'
```

Comment:
this should also be moved under the if statement below, I think

Reply:
It doesn't hurt to set it properly though?
from @jansel comment on another PR, FX does rely on this field to be set to be able to do `{fn.__module__}{fn.__name__}()`.